src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,46 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> sumDigPow(long start, long end) {
 
-		List<Long> result = new ArrayList<Long>();
+		List<Long> results = new ArrayList<Long>();
 
-		for (long i = a; i < b; i++) {
+		for (long currentNumber = start; currentNumber < end; currentNumber++) {
 
-			List<Long> longs = new ArrayList<Long>();
-
-			String temp = Long.toString(i);
-
-			// Split values
-			for (int j = 0; j < temp.length(); j++) {
-				longs.add(Long.valueOf(temp.substring(j, j + 1)));
-			}
-
-			// Create sum
+			List<Long> splitCharacters = new ArrayList<Long>();
+			String currentNumberAsString = Long.toString(currentNumber);
+            
+            splitNumber(splitCharacters, currentNumberAsString);
```

Comment:
it is a better idea to return the splitCaharacter list instead of creating it a a couple of line numbers above